ui/types/vault/models/capabilities.d.ts

```diff
@@ -4,9 +4,9 @@
  */
 
 import ComputedProperty from '@ember/object/computed';
-import Model from '@ember-data/model';
+import { Model } from 'vault/app-types';
 
-interface CapabilitiesModel extends Model {
+type CapabilitiesModel = Model & {
```

Comment:
Out of curiosity - why the syntax change here? From what I know (which is admittedly limited 😅 ) about `type` vs `interface`, either would be suitable in this case

Reply:
I'm not 100% sure but the properties of `Model` like `save` for instance were not getting picked up as belonging to `CapabilitiesModel` when defined as an interface extending `Model`. I'm wondering if it has to do with `Model` being a class and not an interface 🤷? In any case, using `type` with `&` creates a new type with the properties from both.

Reply:
Interesting! Yeah...it must be because `Model` is a class, I think if it were another `interface` it'd extend fine. Cool - thanks for elaborating